tests/bsim/bluetooth/mesh/tests_scripts/proxy_sol/sol_beacon_rcvd.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Test procedure:
+# 1. Initialize and configure tester and iut instances.
+# 3. Tester sends a solicitation PDU.
+# 4. Tester scans for beacons and expects to receive at least one private net ID advertisement.
+conf=prj_mesh1d1_conf
+overlay="overlay_pst_conf_overlay_gatt_conf"
+RunTest proxy_sol_beacon_rcvd \
+	proxy_sol_tester_beacon_rcvd \
+	proxy_sol_iut_beacon_send -flash=../results/mesh_srpl_replay_attack/flash.bin -flash_erase
```

Comment:
If you do not want to restore anything after test rerun, you do not need to provide any options to the test scenario.